### DISCUSSION TITLE
0.19.1：The production build unexpectedly includes jsxdev syntax, leading to runtime errors.

### DISCUSSION THREAD

**Discussion by yebei199** [wxt-dev/wxt#874] [Q&A] [answered]
I'm facing a discrepancy between my development and production builds. The production build unexpectedly includes jsxdev syntax, leading to runtime errors. Although using jiti as a transpiler can mitigate this issue, I'd like to explore other approaches to eliminate the need for jiti.

# use entrypointLoader: 'jiti',
![image](https://github.com/user-attachments/assets/edd66da4-1372-438f-822f-4eecb916c9cf)

# remove entrypointLoader: 'jiti',
![image](https://github.com/user-attachments/assets/5992fb39-a3c9-4b8e-a1b5-89a2e4d22651)

**Comment by 1natsu172** [comment, marked as answer]
@yebei199 The issue with `jsxDev` in production might be the same as this one(#873). Is your error message the same?

**Reply by yebei199** [reply]
yes, I'm sorry for missing this issue. I wasn't feeling well all and my focus was a bit off. To prevent any further disruptions, I'll close this issue for now.

https://github.com/wxt-dev/wxt/issues/873
